### DISCUSSION TITLE
Blacklist words

### DISCUSSION THREAD

**Discussion by kodmani** [night/betterttv#5953] [Ideas]
Can we put in an option to make blacklisted words case sensitive? 
Also, a way to make the word blocked if it's by itself only? It used to be possible long time ago with regex.

**Comment by night** [comment]
You can already do both of these things by leveraging regex matching:

- case sensitive matching: `~/matchtext/`
- matching a word by itself: `~/(^|\s)matchtext($|\s)/`
